### ISSUE TITLE
Horizontal mode does not work properly in mobile devices

### ISSUE THREAD

**Issue by kemkim** [HSLdevcom/digitransit-ui#2100]
When trying to use horizontal mode in mobile devices, list of available bus timetables is not possible to see properly. Solution would be to divide screen to show map on left side of screen and bus timetables on right side of screen.

**Comment by kemkim** [comment]
When using mobile devices in horizontal mode, they should use this way of showing things in screen, similar to desktop devices.
![matkafi smartphone](https://user-images.githubusercontent.com/19935824/34904052-37b3caf4-f846-11e7-985f-8d11b277f574.png)

**Comment by vesameskanen** [comment]
True. The user interface could adapt better to various resolutions and display proportions.

**Comment by meromisi** [comment]
Similar issue has occurred recently. Buttons are scrolled out of view in horizontal mode (usually on iPhone devices). I tested it on iPhone 6s, 8 and 11 and only iPhone 11 passed (I guess due to bigger screen). I've made screenshots and screen records of the bug, where You can see it's hard to get the buttons back in place, after they've scrolled out. (Probably the same issue exists on iPhone 7 and maybe on iPhone SE too).

Tested devices:
iPhone 11 - passed
- iOS 13.4
iPhone 8 - failed
- iOS 13.3.1
iPhone 6s - failed
- iOS 12.1.3 

Screen records:
https://drive.google.com/open?id=1IpGIHmoV1SFtH2FUBDwJdE2dWfRt57Zt

iPhone 8 screenshot:
![InkediPhone8-issue](https://user-images.githubusercontent.com/54352878/81272228-82d37880-904d-11ea-9f08-4a63e8b189eb.jpg)


iPhone 6s screenshot:
![InkediPhone6s-issue](https://user-images.githubusercontent.com/54352878/81272158-66374080-904d-11ea-8faa-328b7c3ee623.jpg)
